frontend/404.html

```diff
@@ -1,33 +1,39 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>Page Not Found</title>
+<html lang="en">
+	<head>
+		<meta charset="utf-8" />
+		<link rel="icon" href="favicon.svg" />
+		<link rel="preconnect" href="https://fonts.googleapis.com" />
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
```

Comment:
I dont really understand why one of these has `crossorigin` and the other does not

Reply:
I don't know either tbh, but when you use a webfont from google, this is the snippet they recommend you copy, e.g.:
https://fonts.google.com/specimen/Press+Start+2P
sidebar on right, `Use on the web` section.